### PR TITLE
LPS-120305 add the new class when browser is IE

### PR DESCRIPTION
**Pull request by marcosapmf**
Related Issue: https://issues.liferay.com/browse/LPS-120305

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 1]
```diff
@@ -288,6 +288,10 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 	Liferay.on('destroyPortlet', clearPortletHandlers);
 
 	Liferay.Forms.App.start();
+
+	if (Liferay.Browser.isIe()) {
+		document.querySelector('.portlet-forms').classList.add('portlet-forms-ie');
+	}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:294]: [Uncle Bob](https://g.co/kgs/jLxjYH) says: "What a clean code, huh?" 😄 🎉

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**